include/pmacc/mappings/kernel/BorderMapping.hpp

```diff
@@ -103,15 +108,14 @@ namespace pmacc
             return result;
         }
 
-        /**
-         * Returns index of current logical block
+        /** Return index of a supercell to be processed by the given alpaka block
          *
-         * @param realSuperCellIdx current SuperCell index (block index)
-         * @return mapped SuperCell index
+         * @param blockIdx alpaka block index
+         * @return mapped SuperCell index including guards
          */
-        HDINLINE DimDataSpace getSuperCellIndex(const DimDataSpace& realSuperCellIdx) const
+        HDINLINE DimDataSpace getSuperCellIndex(const DimDataSpace& blockIdx) const
```

Comment:
This argument was also named incorrectly and misleadingly. The mapper is block -> supercell not vice versa as one might have understood in the old code

Reply:
Yes your renaming at this point is correct.